bazel/thirdparty/seastar.BUILD

```diff
@@ -557,7 +557,7 @@ cc_library(
         "include/seastar/util/variant_utils.hh",
         "include/seastar/websocket/server.hh",
     ],
-    copts = select({
+    copts = ["-U_FORTIFY_SOURCE"] + select({
```

Comment:
nice. i noticed that even though seastar adds this to the public, inherited version of cppflags, longjmp is only used in .cc files. so hopefully this works ok.